src/components/App.js

```diff
@@ -1,21 +1,25 @@
 import React, { useEffect } from 'react';
 import { Route, Switch } from 'react-router-dom';
-import { useDispatch } from 'react-redux';
+import { Provider, useDispatch } from 'react-redux';
 import MyProfile from './MyProfile';
 import Rockets from './Rockets';
 import { loadRocketsThunk } from '../redux/rockets/rockets';
 
 import '../style/App.css';
 import Navbar from './Navbar';
 import Missions from './Missions';
+import store from '../redux/storeConfig';
+import { getMissions } from '../redux/missions/missions';
 
 function App() {
   const dispatch = useDispatch();
   useEffect(() => {
     dispatch(loadRocketsThunk());
   }, [dispatch]);
+
+  store.dispatch(getMissions());
   return (
```

Comment:
[Optional]
You moved your getMissions action into App.js and it doesn't fetch data from API each time you reload the Missions component,  but you didn't put it into useEffect() hook. So it is rendering now twice.  You can see on the screenshot how it renders twice.
![image](https://user-images.githubusercontent.com/63932912/135937246-45dca776-8b28-4380-a2d3-4e554ba1e2e0.png)

I think it would be better if we put both actions together into useEffect() hook and don't make additional fetch request.